extensions/src/main/scala/ai/h2o/sparkling/extensions/rest/api/Paths.scala

```diff
@@ -24,4 +24,8 @@ object Paths {
   val FINALIZE_FRAME: String = "/3/FinalizeFrame"
   val UPLOAD_PLAN: String = "/3/UploadPlan"
   val LOG_LEVEL: String = "/3/LogLevel"
+  val SW_INTERNAL: String = "/3/sw_internal/*"
```

Comment:
nit: Can we use the same naming conventions for path?

Reply:
👍